packages/popups/src/Notice/Notice.tsx

```diff
@@ -7,34 +7,50 @@ import {
     makeStyles,
     TypographyVariants,
     UILabel,
+    UIBoxButtonVariant,
 } from '@tonlabs/uikit.hydrogen';
 import { UIConstant } from '@tonlabs/uikit.navigation';
 import { NoticeProps, UINoticeType, UINoticeColor } from './types';
 import { Action } from './Action';
+import { CountdownCirlce } from './CountdownCircle';
 
 const getBackgroundColor = (color: UINoticeColor, theme: Theme): ColorValue => {
     switch (color) {
         case UINoticeColor.Secondary:
             return theme[ColorVariants.BackgroundSecondary];
         case UINoticeColor.Negative:
-            return theme[ColorVariants.BackgroundNegative];
+            /**
+             * TODO Add new color to ColorVariants
```

Comment:
Will you add it? :)

Reply:
When the designer adds this color to the palette

Reply:
Can you please discuss it with Aleksei and let's add it now, no need to wait for it

Reply:
Done